src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,78 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage = 1,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
+
+  const getTotalNumbersArray = (): number[] => {
+    return getNumbers(1, totalPages);
+  };
+
+  const handlePageChange = (newPage: number) => {
+    if (newPage !== currentPage) {
+      onPageChange(newPage);
+    }
+  };
+
+  return (
+    <ul className="pagination">
+      <li className={cn('page-item', { disabled: currentPage === 1 })}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1}
+          onClick={() => {
+            if (currentPage > 1) {
+              handlePageChange(currentPage - 1);
+            }
+          }}
```

Comment:
create a function `handlePageChange` and move this logic to the function
```suggestion
          onClick={handlePageChange}
```